docs/transfer-ledger-create.html

```diff
@@ -0,0 +1,1092 @@
+                 {
+                   type: 'checkPredicate',
+                   prompt: 'Keyset Predicate is Invalid'
+                 }
+               ]
+             },
+             toAccountGuard: {
+               identifier : 'custom-guard',
+               rules : [
+                 {
+                   type: 'checkCustomKeyLength',
+                   prompt: 'Length does not match a Public Key'
+                 },
+                 {
+                   type: 'checkPredicate',
+                   prompt: 'Keyset Predicate is Invalid'
+                 }
+               ]
+             },
+             existingGuard: {
+               identifier : 'receiver-keyset-fixed',
+               rules : [
+                 {
+                   type: 'checkKeyset',
+                   prompt: 'Transferring to the receiver is not supported.'
+                 }
+               ]
+             },
+             amount: {
+               identifier  : 'amount',
+               rules: [
+                 {
+                   type   : 'decimal[<0]',
+                 }
+               ]
+             }
+           },
+           inline : true,
+           onSuccess(e){
+             e.preventDefault();
+             const {fromAccount, toAccount, fromChainId, toChainId} = getInputs();
+             if (!isCrossChain(fromChainId, toChainId) && (fromAccount === toAccount)){
+               setError("Sender Account is the Receiver Account")
+             } else {
+               clearError();
+               if (formIsValid() && document.getElementById("signature").value.length !== 0){
+                 submit();
+                 document.getElementById('submit-button').classList.add("disabled");
+                 document.getElementById('reset-button').classList.add("disabled");
+               }
+             }
+           }
+         });
+
+      //Activate Menu
+      $('.ui.secondary.pointing.menu').on('click', ".item", function(e) {
+         $('.ui .item').removeClass('active');
+         $(this).addClass('active');
+         $(`.ui .segment .field`).attr("hidden", true)
+         $(`.ui .segment #${e.target.id}-input`).attr("hidden", false)
+       });
+       //Activate Dropdown
+       $('.dropdown')
+         .dropdown();
+
+       $('.message .close')
+         .on('click', function() {
+           $(this)
+             .closest('.message')
+             .transition('fade')
+           ;
+         })
+       ;
+     });
+
+     const mkReq = (cmd) => {
+         return {
+             headers: {
+                 "Content-Type": "application/json"
+             },
+             method: "POST",
+             body: JSON.stringify(cmd)
+         };
+     };
+
+     // INITIATION FUNCTIONS
+     window.addEventListener('load', function (event) {
+       document.getElementById("getSig-button").addEventListener("click", async function(e){
+            dimBody("Waiting for ledger signature");
+            await signWithLedger();
```

Comment:
When i reject on the ledger i see an unhandled exception in the browser console. The dimmer stays on and im unable to try again without reloading the whole page. Maybe we could wrap this in a `try` and hide the dimmer on any exceptions as well?

Reply:
fixed in https://github.com/obsidiansystems/kadena-transfer-js/pull/5